rust/tests/adls_gen2_backend_test.rs

```diff
@@ -1,39 +1,192 @@
 #[cfg(feature = "azure")]
+/// An Azure Data Lake Gen2 Storage Account is required to run these tests and must be provided by
+/// the developer. Because of this requirement, the tests cannot run in CI and are therefore marked
+/// #[ignore]. As a result, the developer must execute these tests on their machine.
+/// In order to execute tests, remove the desired #[ignore] below and execute via:
+/// 'cargo test --features azure --test adls_gen2_backend_test -- --nocapture'
+/// `AZURE_STORAGE_ACCOUNT_NAME` is required to be set in the environment.
```

Comment:
It could be a follow-up: having dedicated Azure storage options would be really helpful if we want to configure the storage from other Bindings.

S3 Storage options: https://github.com/delta-io/delta-rs/blob/2fa81c611e74235c13e5d27813d2e9d815d10ad9/rust/src/storage/s3/mod.rs#L144

S3 Storage options in action with Python binding: https://github.com/delta-io/delta-rs/blob/main/python/docs/source/usage.rst#loading-a-delta-table

Reply:
Yes, I filed an issue for this: #555